content/2017/02/09/generating-documentation-polyglot-projects.rst

```diff
@@ -0,0 +1,123 @@
+public: yes
+author: david
+tags: [
+  Docs,
+  Tools,
+  Javascript,
+  Python,
+  Sass
+  ]
+summary: |
+  Code documentation is ideally written as close to the actual code
+  as possible, but compiled into a comprehensive set of documentation
+  that includes code from all languages in use. Here's how we intend
+  to do that.
+
+
+Generating code documentation for polyglot projects
+===================================================
+
+At OddBird we believe that undocumented code is unfinished code.
+The act of documenting clarifies what we are building,
+and the resulting documentation guides consistency in how we do things
+in different parts of a project built by different team members
+at different times.
+
+In a typical project, one of our documentation deliverables is a
+living styleguide [link?]
```

Comment:
Like QuarqNet https://www.quarqnet.com/styleguide/index.html or do you mean one like Lonely Planet's Rizzo? http://rizzo.lonelyplanet.com/styleguide/design-elements/colours